### PR TITLE
zh-cn/web/html: clear up title of a

### PR DESCRIPTION
**Pull request by fwqaaq**

### CODE REVIEW

**Comment by Jack-Zhang-1314** [comment]
If there is no problem, I will continue.

File: files/zh-cn/web/html/attributes/index.html [hunk 1]
```diff
@@ -311,8 +311,8 @@ <h2 id="属性列表">属性列表</h2>
...
-   <td>Indicates the action of the element, overriding the action defined in the <a href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/form" title="The HTML &lt;form> element represents a document section containing interactive controls for submitting information."><code>&lt;form&gt;</code></a>.</td>
+   <td><a href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input"> element is used to create interactive controls for web-based forms in order to accept data from the user; a wide variety of types of input data and control widgets are available, depending on the device and user agent. "><code>&lt;input&gt;</code></a>, <a href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button"> element represents a clickable button, used to submit forms or anywhere in a document for accessible, standard button functionality."><code>&lt;button&gt;</code></a></td>
+   <td>Indicates the action of the element, overriding the action defined in the <a href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/form"> element represents a document section containing interactive controls for submitting information."><code>&lt;form&gt;</code></a>.</td>
```
Comment (yin1999) [inline files/zh-cn/web/html/attributes/index.html:315]: not right here
```suggestion
   <td><a href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input"><code>&lt;input&gt;</code></a>, <a href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button"><code>&lt;button&gt;</code></a></td>
   <td>Indicates the action of the element, overriding the action defined in the <a href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/form"><code>&lt;form&gt;</code></a>.</td>
```

File: files/zh-cn/web/html/element/abbr/index.html [hunk 2]
```diff
@@ -52,7 +52,7 @@
 
 <h2 id="属性">属性</h2>
 
-<p>这个元素只有<a href="https://developer.mozilla.org/zh-CN/docs/Web/HTML/Global_attributes" style="line-height: 21px;" title="HTML/Global attributes">全局属性</a>。在同 <code>&lt;abbr&gt;</code> 元素一起使用的时候，{{htmlattrxref("title")}} 属性有特殊的语法含义; 它必须包含对一个缩写完全天然可读的扩写或者描述。这些文本通常会作为提示，在鼠标指针覆盖到该元素时显示作为一个提示。</p>
+<p>这个元素只有<a href="https://developer.mozilla.org/zh-CN/docs/Web/HTML/Global_attributes" style="line-height: 21px;">全局属性</a>。在同 <code>&lt;abbr&gt;</code> 元素一起使用的时候，{{htmlattrxref("title")}} 属性有特殊的语法含义; 它必须包含对一个缩写完全天然可读的扩写或者描述。这些文本通常会作为提示，在鼠标指针覆盖到该元素时显示作为一个提示。</p>
```
Comment (yin1999) [inline files/zh-cn/web/html/element/abbr/index.html:55]:
```suggestion
<p>这个元素只有<a href="https://developer.mozilla.org/zh-CN/docs/Web/HTML/Global_attributes">全局属性</a>。在同 <code>&lt;abbr&gt;</code> 元素一起使用的时候，{{htmlattrxref("title")}} 属性有特殊的语法含义; 它必须包含对一个缩写完全天然可读的扩写或者描述。这些文本通常会作为提示，在鼠标指针覆盖到该元素时显示作为一个提示。</p>
```

File: files/zh-cn/web/html/element/input/index.html [hunk 3]
```diff
@@ -770,7 +770,7 @@ <h2 id="CSS">CSS</h2>
 <h3 id="UI_pseudo-classes">UI pseudo-classes</h3>
 
 <table class="standard-table">
- <caption>Captions super relevant to the <a href="/en-US/docs/Web/HTML/Element/input" title="The HTML &lt;input> element is used to create interactive controls for web-based forms in order to accept data from the user; a wide variety of types of input data and control widgets are available, depending on the device and user agent. "><code>&lt;input&gt;</code></a> element:</caption>
+ <caption>Captions super relevant to the <a href="/en-US/docs/Web/HTML/Element/input"> element is used to create interactive controls for web-based forms in order to accept data from the user; a wide variety of types of input data and control widgets are available, depending on the device and user agent. "><code>&lt;input&gt;</code></a> element:</caption>
```
Comment (yin1999) [inline files/zh-cn/web/html/element/input/index.html:773]:
```suggestion
 <caption>Captions super relevant to the <a href="/en-US/docs/Web/HTML/Element/input"><code>&lt;input&gt;</code></a> element:</caption>
```

File: files/zh-cn/web/html/element/input/index.html [hunk 4]
```diff
@@ -779,71 +779,71 @@ <h3 id="UI_pseudo-classes">UI pseudo-classes</h3>
...
+   <td><a href="/en-US/docs/Web/CSS/:checked">), checkbox (&amp;lt;input type="checkbox">), or option (&amp;lt;option> in a &amp;lt;select>) element that is checked or toggled to an on state.'><code>:checked</code></a></td>
```
Comment (yin1999) [inline files/zh-cn/web/html/element/input/index.html:806]:
```suggestion
   <td><a href="/en-US/docs/Web/CSS/:checked"><code>:checked</code></a></td>
```

**Review by yin1999 — CHANGES_REQUESTED** [review]
There are still a lot

**Comment by yin1999** [comment]
you should improve the rule. I have an example here: <https://gist.github.com/yin1999/c868b0b840b5109a0335b54ba4598e96>, but I don't recommend updating the link's locale and update redirects. just run with `-d` arg (and optional `-limit`)

**Comment by Jack-Zhang-1314** [comment]
Sorry, I don't think about it that much, I will refer to your example and keep improving.

**Comment by Jack-Zhang-1314** [comment]
Or I need to wait util tomorrow

File: files/zh-cn/web/html/attributes/index.html [hunk 5]
```diff
@@ -36,7 +36,7 @@ <h2 id="属性列表">属性列表</h2>
   </tr>
   <tr>
    <td><a href="/zh-CN/docs/Web/HTML/Global_attributes/accesskey">accesskey</a></td>
-   <td><a href="/en/HTML/Global_attributes" title="en/HTML/Global attributes">全局属性</a></td>
+   <td><a href="/en/HTML/Global_attributes">全局属性</a></td>
```
Comment (linlinzzo) [inline files/zh-cn/web/html/attributes/index.html:39]:
```suggestion
   <td><a href="/zh-CN/docs/Web/HTML/Global_attributes">全局属性</a></td>
```

File: files/zh-cn/web/html/attributes/index.html [hunk 6]
```diff
@@ -148,7 +148,7 @@ <h2 id="属性列表">属性列表</h2>
   </tr>
   <tr>
    <td><a href="/zh-CN/docs/Web/HTML/Global_attributes/class">class</a></td>
-   <td><a href="/en/HTML/Global_attributes" title="en/HTML/Global attributes">全局属性</a></td>
+   <td><a href="/en/HTML/Global_attributes">全局属性</a></td>
```
Comment (linlinzzo) [inline files/zh-cn/web/html/attributes/index.html:151]:
```suggestion
   <td><a href="/zh-CN/docs/Web/HTML/Global_attributes">全局属性</a></td>
```

File: files/zh-cn/web/html/attributes/index.html [hunk 7]
```diff
@@ -189,12 +189,12 @@ <h2 id="属性列表">属性列表</h2>
   </tr>
   <tr>
    <td><a href="/zh-CN/docs/Web/HTML/Global_attributes/contenteditable">contenteditable</a></td>
-   <td><a href="/en/HTML/Global_attributes" title="en/HTML/Global attributes">全局属性</a></td>
+   <td><a href="/en/HTML/Global_attributes">全局属性</a></td>
```
Comment (linlinzzo) [inline files/zh-cn/web/html/attributes/index.html:192]:
```suggestion
   <td><a href="/zh-CN/docs/Web/HTML/Global_attributes">全局属性</a></td>
```

File: files/zh-cn/web/html/attributes/index.html [hunk 8]
```diff
@@ -189,12 +189,12 @@ <h2 id="属性列表">属性列表</h2>
...
    <td>指出该元素的内容是否可以被编辑。</td>
   </tr>
   <tr>
    <td><a href="/zh-CN/docs/Web/HTML/Attributes/contextmenu">contextmenu</a></td>
-   <td><a href="/en/HTML/Global_attributes" title="en/HTML/Global attributes">全局属性</a></td>
+   <td><a href="/en/HTML/Global_attributes">全局属性</a></td>
```
Comment (linlinzzo) [inline files/zh-cn/web/html/attributes/index.html:197]:
```suggestion
   <td><a href="/zh-CN/docs/Web/HTML/Global_attributes">全局属性</a></td>
```

下面的也一样替换。

File: files/zh-cn/web/html/attributes/index.html [hunk 9]
```diff
@@ -311,8 +311,8 @@ <h2 id="属性列表">属性列表</h2>
...
-   <td>Indicates the action of the element, overriding the action defined in the <a href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/form" title="The HTML &lt;form> element represents a document section containing interactive controls for submitting information."><code>&lt;form&gt;</code></a>.</td>
+   <td><a href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input"> element is used to create interactive controls for web-based forms in order to accept data from the user; a wide variety of types of input data and control widgets are available, depending on the device and user agent. "><code>&lt;input&gt;</code></a>, <a href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button"> element represents a clickable button, used to submit forms or anywhere in a document for accessible, standard button functionality."><code>&lt;button&gt;</code></a></td>
```
Comment (linlinzzo) [inline files/zh-cn/web/html/attributes/index.html:314]:
```suggestion
   <td><a href="/zh-CN/docs/Web/HTML/Element/input"> element is used to create interactive controls for web-based forms in order to accept data from the user; a wide variety of types of input data and control widgets are available, depending on the device and user agent. "><code>&lt;input&gt;</code></a>, <a href="/zh-CN/docs/Web/HTML/Element/button"> element represents a clickable button, used to submit forms or anywhere in a document for accessible, standard button functionality."><code>&lt;button&gt;</code></a></td>
```

File: files/zh-cn/web/html/attributes/index.html (hunk 1 above)
Comment (linlinzzo) [inline files/zh-cn/web/html/attributes/index.html:315]:
```suggestion
   <td>Indicates the action of the element, overriding the action defined in the <a href="/zh-CN/docs/Web/HTML/Element/form"> element represents a document section containing interactive controls for submitting information."><code>&lt;form&gt;</code></a>.</td>
```

File: files/zh-cn/web/html/attributes/index.html [hunk 10]
```diff
@@ -328,7 +328,7 @@ <h2 id="属性列表">属性列表</h2>
  <tbody>
   <tr>
    <td>hidden</td>
-   <td><a href="/en/HTML/Global_attributes" title="en/HTML/Global attributes">全局属性</a></td>
+   <td><a href="/en/HTML/Global_attributes">全局属性</a></td>
```
Comment (linlinzzo) [inline files/zh-cn/web/html/attributes/index.html:331]:
```suggestion
   <td><a href="/zh-CN/docs/Web/HTML/Global_attributes">全局属性</a></td>
```

同上

File: files/zh-cn/web/html/block-level_elements/index.html [hunk 11]
```diff
@@ -42,7 +42,7 @@ <h2 id="块级元素与行内元素">块级元素与行内元素</h2>
  <dd>一般块级元素可以包含行内元素和其他块级元素。这种结构上的包含继承区别可以使块级元素创建比行内元素更”大型“的结构。</dd>
 </dl>
 
-<p>HTML 标准中块级元素和行内元素的区别至高出现在 4.01 标准中。在 HTML5，这种区别被一个更复杂的<a href="/zh-CN/docs/HTML/Content_categories" title="/zh-CN/docs/HTML/Content_categories">内容类别</a>代替。”块级“类别大致相当于 HTML5 中的<a href="/zh-CN/docs/HTML/Content_categories#Flow_content" title="/zh-CN/docs/HTML/Content_categories#Flow_content">流内容</a>类别，而”行内“类别相当于 HTML5 中的<a href="/zh-CN/docs/HTML/Content_categories#Phrasing_content" title="/zh-CN/docs/HTML/Content_categories#Phrasing_content">措辞内容</a>类别，不过除了这两个还有其他类别。</p>
+<p>HTML 标准中块级元素和行内元素的区别至高出现在 4.01 标准中。在 HTML5，这种区别被一个更复杂的<a href="/zh-CN/docs/HTML/Content_categories">内容类别</a>代替。”块级“类别大致相当于 HTML5 中的<a href="/zh-CN/docs/HTML/Content_categories#Flow_content">流内容</a>类别，而”行内“类别相当于 HTML5 中的<a href="/zh-CN/docs/HTML/Content_categories#Phrasing_content">措辞内容</a>类别，不过除了这两个还有其他类别。</p>
```
Comment (linlinzzo) [inline files/zh-cn/web/html/block-level_elements/index.html:45]:
```suggestion
<p>HTML 标准中块级元素和行内元素的区别至高出现在 4.01 标准中。在 HTML5，这种区别被一个更复杂的<a href="/zh-CN/docs/HTML/Content_categories">内容类别</a>代替。”块级“类别大致相当于 HTML5 中的<a href="/zh-CN/docs/HTML/Content_categories#流内容">流内容</a>类别，而”行内“类别相当于 HTML5 中的<a href="/zh-CN/docs/HTML/Content_categories#措辞内容">措辞内容</a>类别，不过除了这两个还有其他类别。</p>
```

@yin1999 这个页面目前还没有中文翻译，要修改锚文本吗？

File: files/zh-cn/web/html/cors_enabled_image/index.html [hunk 12]
```diff
@@ -53,7 +53,7 @@ <h3 id="网站服务器配置">网站服务器配置</h3>
 
 <h3 id="实现保存功能">实现保存功能</h3>
 
-<p>配置完毕后，你就可以将这些格式的图片保存到本地 <a href="/zh-CN/docs/Web/Guide/API/DOM/Storage" title="/zh-CN/docs/Web/Guide/API/DOM/Storage">DOM 存储</a> 中了，就像这些图片在你自己域名之下一样。</p>
+<p>配置完毕后，你就可以将这些格式的图片保存到本地 <a href="/zh-CN/docs/Web/Guide/API/DOM/Storage">DOM 存储</a> 中了，就像这些图片在你自己域名之下一样。</p>
```
Comment (linlinzzo) [inline files/zh-cn/web/html/cors_enabled_image/index.html:56]:
```suggestion
<p>配置完毕后，你就可以将这些格式的图片保存到本地 <a href="/zh-CN/docs/Web/Guide/API/DOM/Storage">DOM 存储</a>中了，就像这些图片在你自己域名之下一样。</p>
```

File: files/zh-cn/web/html/element/a/index.html [hunk 13]
```diff
@@ -21,7 +21,7 @@
 <table class="properties">
  <tbody>
   <tr>
-   <th scope="row"><dfn><a href="/en-US/docs/HTML/Content_categories" title="HTML/Content_categories">内容分类</a></dfn></th>
+   <th scope="row"><dfn><a href="/en-US/docs/HTML/Content_categories">内容分类</a></dfn></th>
```
Comment (linlinzzo) [inline files/zh-cn/web/html/element/a/index.html:24]:
```suggestion
   <th scope="row"><dfn><a href="/zh-CN/docs/Web/Guide/HTML/Content_categories">内容分类</a></dfn></th>
```

File: files/zh-cn/web/html/element/acronym/index.html [hunk 14]
```diff
@@ -17,7 +17,7 @@ <h2 id="Summary">简介</h2>
 
 <h2 id="Attributes">属性</h2>
 
-<p>该元素除了 <a class="new " href="/en-US/docs/HTML/global_attributes" rel="internal" title="HTML/global attributes">global attributes</a>, 所有其他元素的公共属性之外没有其他属性。</p>
+<p>该元素除了 <a class="new " href="/en-US/docs/HTML/global_attributes" rel="internal">global attributes</a>, 所有其他元素的公共属性之外没有其他属性。</p>
```
Comment (linlinzzo) [inline files/zh-cn/web/html/element/acronym/index.html:20]:
```suggestion
<p>该元素除了 <a class="new " href="/zh-CN/docs/Web/HTML/Global_attributes" rel="internal">global attributes</a>, 所有其他元素的公共属性之外没有其他属性。</p>
```

**Review by linlinzzo — COMMENTED** [review]
本 review 中申请修改的链接均已在 MDN 的网站上试验。

File: files/zh-cn/web/html/block-level_elements/index.html (hunk 11 above)
Comment (yin1999) [reply files/zh-cn/web/html/block-level_elements/index.html:45]: not necessary

**Comment by Jack-Zhang-1314** [comment]
* Beacuse of its part of a tag looks like this, `title` consists of a lot of `>`, therefore it's hard to match what is in the a tag.
* And `<a>`  has a little class, ref, style..., do you need to delete them?

>\<a href="/en-US/docs/Web/CSS/:checked" title="The :checked CSS pseudo-class selector represents any radio (&amp;lt;input type='radio'>), checkbox (&amp;lt;input type='checkbox>), or option (&amp;lt;option> in a &amp;lt;select>) element that is checked or toggled to an on state.">\<code>:checked\</code>\</a>

* My implementation is simple;<https://gist.github.com/Jack-Zhang-1314/f3a0a3c411013cbf52124dcf8f0e7587>

File: files/zh-cn/web/html/attributes/crossorigin/index.html [hunk 15]
```diff
@@ -78,5 +78,5 @@ <h3>link crossorigin</h3>
 <h2 id="另请参阅">另请参阅</h2>
 
 <ul>
- <li><a href="/zh-CN/docs/Web/HTTP/Access_control_CORS" title="en/HTTP access control">HTTP access control</a>（HTTP 访问控制）</li>
+ <li><a href="/zh-CN/docs/Web/HTTP/Access_control_CORS" >HTTP access control</a>（HTTP 访问控制）</li>
```
Comment (yin1999) [inline files/zh-cn/web/html/attributes/crossorigin/index.html:81]: trailing space shoud be removed:

```suggestion
 <li><a href="/zh-CN/docs/Web/HTTP/Access_control_CORS">HTTP access control</a>（HTTP 访问控制）</li>
```

It's better to parse the `<a>` tag.

**Review by yin1999 — CHANGES_REQUESTED** [review]

File: files/zh-cn/web/html/attributes/crossorigin/index.html (hunk 15 above)
Comment (yin1999) [reply files/zh-cn/web/html/attributes/crossorigin/index.html:81]: you can refer this:

```go
func parseAnchor(data string) (href, content string, err error) {
	index := strings.Index(data, "href")
	if index == -1 {
		return "", "", fmt.Errorf("cannot find href")
	}
	data = data[index+len(href):]
	endIndex := strings.Index(data, "</a>")
	if endIndex == -1 {
		return "", "", fmt.Errorf("cannot find </a>")
	}
	token := rune(0)
	start, end := 0, 0
	for i, b := range data {
		switch b {
		case '"', '\'':
			if token == 0 {
				token = b
				if start == 0 {
					start = i
				}
			} else if token == b {
				if end == 0 {
					end = i
				}
				token = 0
			}
		case '>':
			if end != 0 && token == 0 {
				href = data[start+1 : end]
				content = data[i+1 : endIndex]
				return
			}
		}
	}
	return "", "", fmt.Errorf("cannot find href")
}
```

Comment (Jack-Zhang-1314) [reply files/zh-cn/web/html/attributes/crossorigin/index.html:81]: Wow, good recommendation

Comment (Jack-Zhang-1314) [reply files/zh-cn/web/html/attributes/crossorigin/index.html]: We only need content of the href in the a tag?

Comment (yin1999) [reply files/zh-cn/web/html/attributes/crossorigin/index.html]: yes

Comment (Jack-Zhang-1314) [reply files/zh-cn/web/html/attributes/crossorigin/index.html]: oh, wait a minute

**Review by yin1999 — APPROVED** [review]

**Pull request merged**